plugins/account_history_plugin/include/eos/account_history_plugin/account_history_object.hpp

```diff
@@ -13,15 +26,18 @@ class transaction_history_object : public chainbase::object<chain::transaction_h
    id_type             id;
    block_id_type       block_id;
    transaction_id_type transaction_id;
+   AccountName         account_name;
```

Comment:
I think I initially collapsed it to avoid the second lookup (transaction id to block id), but I agree now since it will be much smaller (block_id not repeated for every account name in scope and account name only once with vector storage for transaction ids), but in terms of the logic it will just simplify calling equal_range to find and then will need some more logic to find the block_id.  Either way I agree that I need to separate the storage.